cmd/variable.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/azure/spin-aks-plugin/pkg/azure"
+	"github.com/azure/spin-aks-plugin/pkg/logger"
+	"github.com/spf13/cobra"
+)
+
+var kvName string
+var secretName string
+
+func init() {
+	rootCmd.AddCommand(variableCmd)
+	rootCmd.Flags().StringVar(&kvName, "kv-name", "", "Name of the keyvault to use")
+	rootCmd.Flags().StringVar(&secretName, "secret-name", "", "Name of the secret to use")
+}
+
+var variableCmd = &cobra.Command{
+	Use:   "variable",
+	Short: "Manages variables in the Spin AKS config file.",
+	Long:  "Manages variables in the Spin AKS config file.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := cmd.Context()
+		lgr := logger.FromContext(ctx)
+
+
+		subscriptionId := ""
+		resourceGroup := ""
```

Comment:
imo logic here should be 

1. validate config (we should add a validate to the config package). If it's invalid prompt them to run spin aks init to fill it in
2. Prompt for kv if it's not already in the config. kv isn't required so it's not bundled into spin aks init

i'm cool if we change that and add it later tho with this just being a proof of concept

Reply:
added kv check to init and validated non-empty fields in store with usererror